src/nile/core/cairo1/common.py

```diff
@@ -0,0 +1,7 @@
+"""Cairo1 common."""
```

Comment:
i think we can remove this file and just replace variables in good ol' `common/` if we drop Cairo0 support